homeassistant/components/tts/__init__.py

```diff
@@ -387,7 +391,16 @@ class Provider(object):
     """Represent a single provider."""
 
     hass = None
-    language = None
+
+    @property
+    def language(self):
+        """Default language."""
+        return None
+
+    @property
+    def supported_languages(self):
+        """List of supported languages."""
+        return None
```

Comment:
Actually since all platforms must override both `language` and `supported_languages` maybe throw exception in both of them?